dlrover/python/master/elastic_training/net_topology.py

```diff
@@ -27,6 +28,7 @@ class NodeTopologyMeta(object):
 
     def __repr__(self) -> str:
         d: Dict[str, Any] = {}
+        d["node_id"] = self.node_id
```

Comment:
Use constant replace these variable.

Reply:
The method `__repr__` is not used and I have removed it.